src/lib/web-component/filter-timestamp/filter-timestamp-element.js

```diff
@@ -163,8 +158,8 @@ export class FilterTimestamp extends HTMLElement {
   }
 
   disconnectedCallback() {
-    browser.menus.onClicked.removeListener(this.setFilterRange);
-    browser.menus.onHidden.removeListener(this.onHiddenListener);
+    browser.menus?.onClicked.removeListener(this.setFilterRange);
+    browser.menus?.onHidden.removeListener(this.onHiddenListener);
```

Comment:
Sorry, I haven't noticed it carefully. Thank you for noticing. We shouldn't unsubscribe any of the listeners when we aren't subscribe to any.

In `connectedCallback()` we are subscribing all the listeners only when the `menus` API is available. I think we should do the same in case of unsubscribing the listeners in `disconnectedCallback()` i.e. we shouldn't unsubscribe to any listeners if  `menus` API is missing. I have done this in d3fa9e1.